crates/core/migrations/2022-01-21-232334_add_column_storefronts_banner_url/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+alter table storefronts
+add column banner_url text not null default '';
```

Comment:
change to default column to '' string.

Reply:
Is this preferable to dropping the `NOT NULL` constraint?  I think having the column be nullable (at least as a temporary measure) would cause fewer logic errors.